src/style.css

```diff
@@ -1 +1,88 @@
-/* Styles go here */
+html {
+  background: hsl(0, 0%, 90%);
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: rgb(255, 255, 255);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 50px;
+  max-width: 100%;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  list-style: none;
+  justify-content: flex-end;
```

Comment:
justify-content: flex-end; should be applied for blocks with display: flex property